pinc/Project.inc

```diff
@@ -814,7 +819,8 @@ class Project
     // by joining them with the string " with ". These functions provide a
     // common place for encoding / decoding them.
 
-    public static function encode_languages($languages)
+    /** @param array{0: string, 1?: string} $languages */
+    public static function encode_languages($languages): string
```

Comment:
```suggestion
    public static function encode_languages(array $languages): string
```
This would then obviate the `is_array()` check at the beginning of the function.

Reply:
Committed the suggestion and removed the `is_array()` check below.